src/lib/platform/EiScreen.cpp

```diff
@@ -695,10 +695,18 @@ void EiScreen::on_motion_event(ei_event* event)
          }
 #endif
     } else {
-        LOG_DEBUG("on_motion_event on secondary at (dx,dy)=(%.2f,%.2f)", dx, dy);
+        buffer_dx += dx;
+        buffer_dy += dy;
+        LOG_DEBUG("on_motion_event on secondary at (dx,dy)=(%.2f,%.2f)", buffer_dx, buffer_dy);
         send_event(EventType::PRIMARY_SCREEN_MOTION_ON_SECONDARY,
-                   create_event_data<MotionInfo>(MotionInfo{static_cast<std::int32_t>(dx),
-                                                            static_cast<std::int32_t>(dy)}));
+                   create_event_data<MotionInfo>(MotionInfo{static_cast<std::int32_t>(buffer_dx),
+                                                            static_cast<std::int32_t>(buffer_dy)}));
```

Comment:
this looks like the `send_event()` should be inside a check for `buffer >= 1`, otherwise you may send a 0/0 motion event.

Reply:
It was the original behaviour, that is why I didn't change it previously, it should be fixed now